packages/components/src/components/Drawer/Drawer.test.tsx

```diff
@@ -0,0 +1,50 @@
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import React from "react";
+import { Default } from "./Drawer.stories";
+
+const OPEN_DRAWER_TEXT = "Open drawer";
+
+describe("<Drawer />", () => {
+  it("should open and close a drawer using the close drawer button", async () => {
+    render(<Default />);
```

Comment:
Why use the component from the Stories? Shouldn't we use the component that we created in the `Drawer.tsx`? 🤔

Reply:
Using the component from stories allows us to test without using duplicate code. Storybook allows stories to be used for testing now, so we don't have to build another component just for tests. Here's an [article](https://storybook.js.org/docs/react/writing-tests/importing-stories-in-tests) for more information.